react/features/prejoin/components/native/Prejoin.tsx

```diff
@@ -97,6 +99,14 @@ const Prejoin: React.FC<IPrejoinProps> = ({ navigation }: IPrejoinProps) => {
     }, [ dispatch, hasDisplayName, isDisplayNameMissing, onJoin ]);
 
     const onJoinLowBandwidth = useCallback(() => {
+
+        // Authentication navigation to the external url resets the state
+        if (isTokenAuthEnabled(config)) {
```

Comment:
This is kinda a workaround. The auth service should retain the startAudioOnly parameter. @damencho does it do that now? Will it retain any hash parameters we pass?

Reply:
@saghul It does retain. You were right. I updated the PR and tested it.